service/discovery/README.md

```diff
@@ -0,0 +1,182 @@
+# Discovery Status
+
+✅: Discovered
+❌: Not Discovered
+🚫: Not available
+
+## Compute
+<details>
+<summary>Compute</summary>
```

Comment:
Is it possible to that with the headline? So we have only one Compute and not one above the other.